core/invoker/src/main/scala/whisk/core/containerpool/DockerContainer.scala

```diff
@@ -162,6 +162,9 @@ class DockerContainer(id: ContainerId, ip: ContainerIp)(
                 transid.finished(this, start.copy(start = r.interval.start), s"initialization result: ${r.toBriefString}", endTime = r.interval.end)
             case Failure(t) =>
                 transid.failed(this, start, s"initializiation failed with $t")
+        }.recoverWith {
+            case t =>
+                Future.failed(InitializationError(ActivationResponse.whiskError("action failed to initialize"), Interval(Instant.EPOCH, Instant.EPOCH)))
```

Comment:
As discussed in Slack: This piece of code should never be reached, as `http.post` shouldn't throw. Being defensive nonetheless, thus reaching this code means a `WhiskError`.

Will add a comment clarifying that.